gemrb/plugins/SDLVideo/SDL20GLVideo.cpp

```diff
@@ -555,12 +555,18 @@ void GLVideoDriver::BlitTile(const Sprite2D* spr, const Sprite2D* mask, int x, i
 	Region dst(tx + dx, ty + dy, w, h);
 
 	const Color* totint = NULL;
+	Color tileTint;
+
 	if (core->GetGame()) {
 		totint = core->GetGame()->GetGlobalTint();
+		if (totint) {
+			tileTint = *totint;
+			tileTint.a = 0xFF;
+		}
```

Comment:
The software renderer doesn't know alpha-tinting for tiles, a neutralizing this value will help us in the shader.